package.json

```diff
@@ -85,9 +85,12 @@
     "prettier-eslint-cli": "5.0.1",
     "semantic-release": "^19.0.5",
     "ts-jest": "^27.0.4",
-    "ts-node": "^8.10.2",
+    "ts-node": "^10.9.1",
     "typescript": "^4.1.3"
   },
+  "resolutions": {
+    "ipfs-unixfs": "6.0.6"
```

Comment:
Why does this need to be pinned?

Reply:
When I created a separate starter project with subql, it had it in there so I added it here as well

Reply:
Do we need overrides and pnpm like the SDK?

I think just yarn support is fine, although not needing it would be best